### PR TITLE
test(PopoverView): Add tests for popover

### PR DESCRIPTION
**Pull request by jmpg93**
I'm just adding some test to the PopoverView. I've noticed that the image get stretched if the text is too short.
I've added a content mode to the image so it does not get deformed.

### CODE REVIEW

File: Mistica/Tests/MisticaTests/UI/PopoverViewTests.swift [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+import Mistica
+import SnapshotTesting
+import XCTest
+
+final class PopoverViewTests: XCTestCase {
+    override class func setUp() {
+        super.setUp()
+
+        isRecording = true
```
Comment (DavidMarinCalleja) [inline Mistica/Tests/MisticaTests/UI/PopoverViewTests.swift:17]:
```suggestion
        isRecording = false
```
maybe you forgot it

Comment (jmpg93) [reply Mistica/Tests/MisticaTests/UI/PopoverViewTests.swift:17]: Wuuuups!

File: Mistica/Tests/MisticaTests/UI/PopoverViewTests.swift [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+        assertSnapshot(
+            matching: popover,
+            as: .image(size: .init(width: 200, height: 100))
+        )
+    }
+
+    // MARK: - Close
+
+    func testNoCloe() {
```
Comment (DavidMarinCalleja) [inline Mistica/Tests/MisticaTests/UI/PopoverViewTests.swift:112]:
```suggestion
    func testNoClose() {
```

**Review by DavidMarinCalleja — APPROVED** [review]
Thanks for the tests

**Review by victorpimentel — APPROVED** [review]

**Pull request merged**